go/vt/vtgate/buffer/variables.go

```diff
@@ -48,26 +60,44 @@ var (
 	// utilization). The moving average would be 100% because there were two
 	// failovers in that period.
 	// TODO(mberlin): Replace this with a MultiHistogram once it's available.
-	utilizationDryRunSum = stats.NewMultiCounters("BufferUtilizationDryRunSum", []string{"Keyspace", "ShardName"})
+	utilizationDryRunSum = stats.NewMultiCounters(
+		"BufferUtilizationDryRunSum",
+		"Cumulative sum of the max *theoretical* buffer util (in %) during each failover",
+		[]string{"Keyspace", "ShardName"})
 
 	// requestsBuffered tracks how many requests were added to the buffer.
 	// NOTE: The two counters "Buffered" and "Skipped" should cover all requests
 	// which passed through the buffer.
-	requestsBuffered = stats.NewMultiCounters("BufferRequestsBuffered", []string{"Keyspace", "ShardName"})
+	requestsBuffered = stats.NewMultiCounters(
+		"BufferRequestsBuffered",
+		"How many requests were added to the buffer",
+		[]string{"Keyspace", "ShardName"})
 	// requestsBufferedDryRun tracks how many requests would have been added to
 	// the buffer (dry-run mode).
-	requestsBufferedDryRun = stats.NewMultiCounters("BufferRequestsBufferedDryRun", []string{"Keyspace", "ShardName"})
+	requestsBufferedDryRun = stats.NewMultiCounters(
+		"BufferRequestsBufferedDryRun",
+		"How many requests would've been added to the buffer (dry-run)",
+		[]string{"Keyspace", "ShardName"})
 	// requestsBuffered tracks how many requests were drained from the buffer.
 	// NOTE: The sum of the two counters "Drained" and "Evicted" should be
 	// identical to the "Buffered" counter value.
-	requestsDrained = stats.NewMultiCounters("BufferRequestsDrained", []string{"Keyspace", "ShardName"})
+	requestsDrained = stats.NewMultiCounters(
+		"BufferRequestsDrained",
+		"How many requests drained from the buffer",
+		[]string{"Keyspace", "ShardName"})
 	// requestsEvicted tracks how many requests were evicted early from the buffer.
 	// See the type "evictedReason" below for all possible values of "Reason".
-	requestsEvicted = stats.NewMultiCounters("BufferRequestsEvicted", []string{"Keyspace", "ShardName", "Reason"})
+	requestsEvicted = stats.NewMultiCounters(
+		"BufferRequestsEvicted",
+		"How many requests were evicted early from the buffer",
+		[]string{"Keyspace", "ShardName", "Reason"})
 	// requestsSkipped tracks how many requests would have been buffered but
 	// eventually were not (includes dry-run bufferings).
 	// See the type "skippedReason" below for all possible values of "Reason".
-	requestsSkipped = stats.NewMultiCounters("BufferRequestsSkipped", []string{"Keyspace", "ShardName", "Reason"})
+	requestsSkipped = stats.NewMultiCounters(
+		"BufferRequestsSkipped",
+		"How many requests would've been buffered but were skipped (incl. dry-run)",
```

Comment:
Minor nit -- I feel like "would have" as opposed to "would've" sounds... more professional :)